### DISCUSSION TITLE
Any documentation about adding new language to RHVoice

### DISCUSSION THREAD

**Discussion by cyrmax** [RHVoice/RHVoice#723] [Q&A]
Hello!
I want to add hungarian language to RHVoice.
I have a native speaker who wants to help; I have time and patience but I really do not have any documentation.
So I ask if someone could give me some information about available options when I fill in phonemes.xml and labelling.xml?
Also, if someone already has some experience in creating new language modules, any advice and recommendations would be very appreciated.
Thanks in advance.

**Comment by louderpages** [comment]
It is a big learning curve to create a new language, even if there were good documentation. 

The big problem is mastering Finite State Transducers, by using the FOMA interpreter.  Have you looked at the FOMA code in some other language here?  

But, you ask specifically about phonemes.xml and labelling, it makes me wonder if perhaps you have learned all about FOMA. :)  Have you?

If you have, I might be able to help you with the phonemes.xml
